.github/scripts/intake_to_linear.py

```
@@ -0,0 +1,161 @@
+def main():
+    s = json.loads(os.environ["STRUCTURED"])
+    repo = os.environ.get("REPO", "")
+    src_url = os.environ.get("REQ_URL", "")
+    src_title = os.environ.get("REQ_TITLE", "")
+
+    # --- resolve team G + its labels ---
+    team = gql('{ teams(filter:{key:{eq:"G"}}){ nodes{ id labels{ nodes{ id name parent{ name } } } } } }')
+    tnode = team["teams"]["nodes"][0]
+    team_id = tnode["id"]
+    label_index = {}
+    for l in tnode["labels"]["nodes"]:
```

Ruff (0.15.18)

[error] Ambiguous variable name: `l`

(E741)